.github/workflows/validate.yaml

```diff
@@ -0,0 +1,29 @@
+name: Lint and Test Charts
+on:
+  pull_request:
+    branches:
+      - dev-v2.6
+
+jobs:
+  validate-chart-questions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Validate all charts have a questions file
```

Comment:
can we move this to scripts, developers should be able to run all validation locally as well.

Reply:
talked offline, given the LOE to add done and current plans for this repo this makes sense to put here

Reply:
We discussed this offline and decided it is better to keep as is since system charts don't have a CI and it would be too much effort for little benefit.